bin/service-status.py

```diff
@@ -83,7 +83,7 @@ def get_config():
 
 
 def fetch(url, is_json=True):
-    resp = requests.get(url)
```

Comment:
This file isn't maintained in this repo. It's maintained in https://github.com/willkg/socorro-release/ . We can make this change here, but we should make it upstream and copy it over at some point. I can take that on if you like--it's drudgery work.

Reply:
filed